compiler/src/typed/parmatch.re

```diff
@@ -967,11 +969,18 @@ let build_other = (ext, env) =>
       p,
       env,
     )
-  /*| ({pat_desc=(TPatConstant (Const_bool _))} as p,_) :: _ ->
-    build_other_constant
-      (function TPatConstant(Const_bool i) -> i | _ -> assert false)
-      (function i -> TPatConstant(Const_bool i))
-      ??? p env*/
+  | [({pat_desc: TPatConstant(Const_bool(_))} as p, _), ..._] =>
```

Comment:
the ol' uncomment and fix an issue! 🔥

Reply:
Haha, this one was clear that we didn't really know how to implement it 😛 

But we do now!